morpheushelper/cogs/reddit.py

```diff
@@ -14,6 +14,7 @@
 from models.reddit import RedditPost, RedditChannel
 from permissions import Permission
 from util import send_to_changelog
+import logging
```

Comment:
perhaps you should consider using the `getLogger` method of logging to get a specific logger and also create a logging config.
```python
logger = logging.getLogger(<name>)
```